smac/utils/util_funcs.py

```diff
@@ -16,6 +16,11 @@ def get_types(config_space, instance_features=None):
             types[i] = n_cats
             bounds[i] = (int(n_cats), np.nan)
 
+        elif isinstance(param, (OrdinalHyperparameter)):
+            n_cats = len(param.sequence)
```

Comment:
ordinals are treated in the same way as categoricals?
This would be wrong because the ordering is indeed important---that's the whole point of ordinals compared to categoricals.